public/static/locales/en/profile.json

```diff
@@ -18,6 +18,7 @@
     },
     "myForestMap": {
       "plantedTree": "{count, plural, =1 {# Tree} other {# Trees}}",
+      "savedArea": "{area} m²",
```

Comment:
May not be needed as you can use "areaContributed"

Reply:
We don't need to show `conserved` and `restored` keyword hence I didn't use `areaContributed`